app/models/manageiq/providers/inventory/persister/builder/cloud_manager.rb

```diff
@@ -89,6 +89,16 @@ def orchestration_stack_ancestry
           )
         end
 
+        def resource_groups
+          # NOTE: This model is different in that it isn't namespaced under a specific
+          # manager (e.g. ManageIQ::Providers::Vendor::CloudManager::ResourceGroup) but
+          # rather directly under the vendor.
+          model_class = "#{manager_class&.module_parent}::ResourceGroup".safe_constantize
```

Comment:
I think this was done under the false belief that in order to reference a model from a manager it couldn't be namespaced under a different manager.

This should live under the CloudManager but will require a data migration to fixup existing records.

Reply:
so this is work for a future set of PRs, right?

Reply:
Correct, I'm working on a data migration for Openstack not subclassing OrchestrationStacks right now, I'll tackle this next